omnigibson/__init__.py

```diff
@@ -71,6 +74,10 @@
 example_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "examples")
 example_config_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "configs")
 
+# Create and expose a temporary directory for any use cases. It will get destroyed *after* omni
+# shutdown by the _kill_omni function.
+tempdir = tempfile.mkdtemp()
```

Comment:
On Linux the decrypted file will immediately get deleted after load so there shouldn't be an issue there.
On windows however, Isaac Sim won't let go of those usd files so we can only delete it upon shutdown, so I guess it should be put here so that og knows what the folder path is?

Reply:
hmmm I see. @ChengshuLi what do you think? will turbosquid folks get mad at us if we do this? lol